study.md

```diff
@@ -21,18 +21,22 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+In the JSIS article on callbacks, there is an example where the function `successCallback()` is defined twice. How does this work?
```

Comment:
It looks like the author copied and pasted that block of code and forgot to rename the first one to something else.